src/bootloader/jira.js

```diff
@@ -6,37 +6,45 @@ var observer = require('../utils/observer');
 var buttons = new HashMap();
 
 function createObserver() {
-  observer.create('.issue-header-content')
+  observer.create('#ghx-detail-view')
     .onAdded(createButton)
     .onRemoved(removeButton)
     .start();
 }
 
 function createButton(node) {
-  var titleEl = node.querySelector('h1');
-  var linkEl = node.querySelector('.issue-link');
-
-  var name = titleEl.innerText;
-  var link = linkEl.href;
-
   var state = new ButtonState({
-    link: link,
     task: {
-      name: name,
-      notes: 'Added from JIRA: ' + link
+      notes: 'Added from JIRA: ' + location.href
     },
-    anchor: 'element'
+    link: location.href,
```

Comment:
When opening the project view the button is already created, but the URL references to the big project list.